File: src/main/java/pl/mjedynak/idea/plugins/pit/cli/PitCommandLineArgumentsContainer.java
package pl.mjedynak.idea.plugins.pit.cli;

import pl.mjedynak.idea.plugins.pit.cli.model.PitCommandLineArgument;

public interface PitCommandLineArgumentsContainer {

    void put(PitCommandLineArgument argument, String value);

    String get(PitCommandLineArgument argument);
}

File: src/main/kotlin/pl/mjedynak/idea/plugins/pit/cli/PitCommandLineArgumentsContainer.kt
package pl.mjedynak.idea.plugins.pit.cli

import pl.mjedynak.idea.plugins.pit.cli.model.PitCommandLineArgument

interface PitCommandLineArgumentsContainer {
    fun put(
        argument: PitCommandLineArgument,
        value: String,
    )

    fun get(argument: PitCommandLineArgument): String?
}
